Content.Server/Atmos/Piping/Unary/EntitySystems/GasThermoMachineSystem.cs

```diff
@@ -89,8 +92,13 @@ private void OnThermoMachineUpdated(EntityUid uid, GasThermoMachineComponent the
                 scale = dQLim / dQ; // reduce power consumption
                 thermoMachine.HysteresisState = false; // turn off
             }
-            float dQActual = dQ * scale;
+            float dQActual = dQ * scale * (1 - thermoMachine.EnergyLeakPercentage);
+            var dQLeak = dQ * scale * thermoMachine.EnergyLeakPercentage;
             _atmosphereSystem.AddHeat(inlet.Air, dQActual);
+
+            if (_atmosphereSystem.GetContainingMixture(uid) is { } containingMixture)
```

Comment:
so it just voids energy if there's no atmosphere to leak heat into? i mean, okay, since this can be explained by it leaking into space, but might be worth considering

Reply:
it would have no downside if being in space disabled venting

Reply:
i'm fine with that or else putting it in space would just give it a defacto boost in efficiency